2021-12-03 12:05:28,014 INFO    MainThread:28600 [wandb_setup.py:_flush():71] setting env: {}
2021-12-03 12:05:28,014 INFO    MainThread:28600 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-12-03 12:05:28,014 INFO    MainThread:28600 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2021-12-03 12:05:28,015 INFO    MainThread:28600 [wandb_init.py:_log_setup():371] Logging user logs to /sites/Garbage_detector/detector/wandb/run-20211203_120528-27lt0ypa/logs/debug.log
2021-12-03 12:05:28,015 INFO    MainThread:28600 [wandb_init.py:_log_setup():372] Logging internal logs to /sites/Garbage_detector/detector/wandb/run-20211203_120528-27lt0ypa/logs/debug-internal.log
2021-12-03 12:05:28,015 INFO    MainThread:28600 [wandb_init.py:init():404] calling init triggers
2021-12-03 12:05:28,015 INFO    MainThread:28600 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-03 12:05:28,015 INFO    MainThread:28600 [wandb_init.py:init():449] starting backend
2021-12-03 12:05:28,015 INFO    MainThread:28600 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-03 12:05:28,554 INFO    MainThread:28600 [backend.py:ensure_launched():199] starting backend process...
2021-12-03 12:05:28,570 INFO    MainThread:28600 [backend.py:ensure_launched():204] started backend process with pid: 28609
2021-12-03 12:05:28,581 INFO    MainThread:28600 [wandb_init.py:init():458] backend started and connected
2021-12-03 12:05:28,582 INFO    MainThread:28600 [wandb_init.py:init():519] updated telemetry
2021-12-03 12:05:28,608 INFO    MainThread:28600 [wandb_init.py:init():549] communicating current version
2021-12-03 12:05:29,258 INFO    MainThread:28600 [wandb_init.py:init():554] got version response 
2021-12-03 12:05:29,258 INFO    MainThread:28600 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-03 12:05:29,734 INFO    MainThread:28600 [wandb_init.py:init():592] starting run threads in backend
2021-12-03 12:05:29,758 INFO    MainThread:28600 [wandb_run.py:_console_start():1816] atexit reg
2021-12-03 12:05:29,758 INFO    MainThread:28600 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-03 12:05:29,759 INFO    MainThread:28600 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-03 12:05:29,760 INFO    MainThread:28600 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-03 12:05:29,760 INFO    MainThread:28600 [wandb_init.py:init():619] run started, returning control to user process
2021-12-03 12:05:29,761 INFO    MainThread:28600 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-12-03 12:05:40,562 INFO    MainThread:28600 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-03 12:05:40,562 INFO    MainThread:28600 [wandb_run.py:_restore():1758] restore
2021-12-03 12:05:42,945 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 685
}

2021-12-03 12:05:43,620 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 685
}

2021-12-03 12:05:44,044 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 685
}

2021-12-03 12:05:44,746 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 4640
}

2021-12-03 12:05:44,848 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 4640
}

2021-12-03 12:05:44,949 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 685
  total_bytes: 4640
}

2021-12-03 12:05:45,052 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1971
  total_bytes: 4640
}

2021-12-03 12:05:45,154 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,256 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,357 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,459 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,561 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,663 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:45,866 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}

2021-12-03 12:05:46,410 INFO    MainThread:28600 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4640
  total_bytes: 4640
}
local_info {
}

2021-12-03 12:05:47,499 INFO    MainThread:28600 [wandb_run.py:_append_files():2184] logging synced files
